BootcampKotlin/app/src/main/java/com/github/ybecker/bootcampKotlin/GreetingActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.github.ybecker.bootcampKotlin
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+
+class GreetingActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
```

Comment:
As in MainActivity.kt you could have written it on one single line, even if it is a detail.